include/veriblock/blob.hpp

```diff
@@ -31,13 +31,13 @@ struct Blob {
   Blob(const Blob<N>& other) : data_(other.data_) {}
   Blob(Blob<N>&& other) noexcept : data_(std::move(other.data_)) {}
 
-  iterator begin() noexcept { return data_.begin(); }
+  iterator begin() noexcept { return data_.data(); }
 
-  const_iterator begin() const noexcept { return data_.begin(); }
+  const_iterator begin() const noexcept { return data_.data(); }
 
-  iterator end() noexcept { return data_.end(); }
+  iterator end() noexcept { return data_.data() + N; }
 
-  const_iterator end() const noexcept { return data_.end(); }
+  const_iterator end() const noexcept { return data_.data() + N; }
```

Comment:
why did you change all this?

Reply:
because data.begin() it is an ArrayIterator type and our 'iterator' is value_type*